### DISCUSSION TITLE
Swipe to next picture when zoomed in

### DISCUSSION THREAD

**Discussion by laurenspriem** [ente-io/ente#1852] [Enhancements]
Feedback from Discord:

> Can you please add a feature to be able to swipe to the next picture when zoomed in? Now it only works in memories, it would be really nice if it worked everywhere.

**Comment by eagle-zero** [comment]
Apple photos could be used as inspiration where you have some resistance to go to the next photo when you have zoomed in.

**Reply by wololz** [reply]
I think it's inconvenient and not very intuitive that you can't switch to the next photo when zoomed in. 

This also causes a odd behavior. When you zoomed in a photo and then view the Live Photo, then you cannot swipe to the next photo. Even though playing the Live Photo automatically zooms out.

You need to zoom back into the photo so that you can zoom out again. Only then can you swipe to the next image.
